### PR TITLE
O3-1169: Implementer tools should be dark themed

### PR DESCRIPTION
**Pull request by joachim-junior**
## Requirements
- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

#### For changes to apps
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

## Summary
This Issue was aimed and changing the theme of the implementer tools app to dark. I also adjusted some little styling like config buttons for example.
## Screenshots
https://user-images.githubusercontent.com/40958761/165086353-b8db0cef-213c-45dc-ac05-1a5ab71fdd8c.mp4

## Related Issue
https://issues.openmrs.org/browse/O3-1169

### CODE REVIEW

**Comment by vasharma05** [comment]
Hi @joachim-junior!
Good work, somethings I have noticed are:

1. The hover on the package name doesn't change the text color to black, hence there's a problem reading it.
2. I can't see the edit and revert icons along with the values in the new design.

Please have a look at it.
Thanks!

**Review by vasharma05 — CHANGES_REQUESTED** [review]
Hey @joachim-junior!
The color theme is not at all in compliance with the designs.
Please refer to the designs in the link: https://wiki.openmrs.org/display/projects/Implementer+Tools+UI+-+Project+Plan.

Please have a look at the designs provided on the page.

File: packages/apps/esm-implementer-tools-app/src/backend-dependencies/backend-dependencies.styles.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "../implementer-tools.styles.scss";
+
+.table {
+  background-color: #000;
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/src/backend-dependencies/backend-dependencies.styles.scss:4]: Please use variables from "~@openmrs/esm-styleguide/src/vars"

File: packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx [hunk 2]
```diff
@@ -162,38 +162,56 @@ export const Configuration: React.FC<ConfigurationProps> = () => {
             <Row>
               <Column sm={1} md={2}>
                 <TextInput
+                  className={styles.text}
                   id="extensionSearch"
-                  labelText="Search configuration"
+                  placeholder="Search config"
+                  labelText={
+                    <p style={{ color: "#efefef" }}>{"Search configuration"}</p>
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx:169]: Please do not use inline styling.

File: packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx [hunk 3]
```diff
@@ -162,38 +162,56 @@ export const Configuration: React.FC<ConfigurationProps> = () => {
...
+                  }
                   onChange={(e) => setFilterText(e.target.value)}
                 />
               </Column>
               <Column sm={1} md={1}>
                 <Toggle
                   id="jsonModeSwitch"
-                  labelText={t("jsonEditor", "JSON Editor")}
+                  labelText={
+                    <p style={{ color: "#efefef" }}>
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx:178]: No inline styling please.

File: packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx [hunk 4]
```diff
@@ -162,38 +162,56 @@ export const Configuration: React.FC<ConfigurationProps> = () => {
...
+                    </p>
+                  }
                   toggled={isUIEditorEnabled}
                   onToggle={toggleIsUIEditorEnabled}
                 />
               </Column>
               <Column sm={1} md={1} className={styles.actionButton}>
                 <Button
                   kind="danger"
+                  className={styles.btn}
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/src/configuration/configuration.component.tsx:213]: Please provide more explicit class names.,

File: packages/apps/esm-implementer-tools-app/src/configuration/json-editor/json-editor.component.tsx [hunk 5]
```diff
@@ -3,9 +3,11 @@ import {
   temporaryConfigStore,
   useStore,
 } from "@openmrs/esm-framework/src/internal";
-import AceEditor from "react-ace";
+import Editor from "react-ace";
+// import { JsonEditor as Editor } from "jsoneditor-react";
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/src/configuration/json-editor/json-editor.component.tsx:7]: Please remove the comments

File: packages/apps/esm-implementer-tools-app/webpack.config.js [hunk 6]
```diff
@@ -1 +1,15 @@
 module.exports = require("openmrs/default-webpack-config");
+// module.exports = {
+//   module: {
```
Comment (vasharma05) [inline packages/apps/esm-implementer-tools-app/webpack.config.js:3]: Please remove the comments

**Review by vasharma05 — CHANGES_REQUESTED** [review]

**Review by joachim-junior — COMMENTED** [review]
I have added some colors in the esm-style-guide in accordance with design of implementer tools 2.0.

**Comment by vasharma05** [comment]
@joachim-junior, can you please upload a video showing all the latest changes.

**Comment by joachim-junior** [comment]
https://user-images.githubusercontent.com/40958761/166897421-c2d425cf-7e97-4a56-aa08-b1e00d031239.mp4

**Comment by brandones** [comment]
For anyone else reading in the future: the above MP4 video displays correctly in Chrome, not Firefox.

@joachim-junior Did you look at the designs? Your implementation looks nothing like them. I attached a PNG of the design to [the ticket](https://issues.openmrs.org/browse/O3-1169).

**Pull request closed without merging**